### ISSUE TITLE
Add brushes to alphaTex

### ISSUE THREAD

**Issue by Danielku15** [CoderLine/alphaTab#505]
Brush strokes can currently not be defined in alphaTex. This should be possible as beat effect. 
Proposed syntax:
`bu {duration}` for brush up
`bd {duration}` for brush down
`au {duration}` for arpeggio up
`ad {duration}` for arpeggio down

The duration is in midi ticks optional and defaults to some calculated value based on the beat duration.

**Comment by goodgame365** [comment]
Has it come true,NOW?

**Comment by Danielku15** [comment]
@goodgame365 If you have a question, can you please try to rephrase it to make it more understandable?

**Comment by jonaro00** [comment]
To clarify: Would the syntax be `(0.1 1.2 0.3 2.4 3.5){ch "C" bd 60}` for example?
